src/ui/Facet/Facet.ts

```diff
@@ -776,6 +776,19 @@ export class Facet extends Component {
     });
   }
 
+  public set currentFacetSearchElementId(id: string) {
+    if (!this.searchContainer) {
+      return;
+    }
+    const { accessibleElement } = this.searchContainer;
+    if (!id) {
+      accessibleElement.removeAttribute('aria-controls');
```

Comment:
Should we assert that the `id` is a non-empty string instead? Could it also be more "fool-proof" to check that there actually is a (child?) element with the id?


Also, as far as removing `aria-controls`, if the `searchContainer` is removed from the DOM when the facet search is closed, do we still need to remove the attribute?

Reply:
All good points. I'll add an assertion to check that `id` is non-empty and exists, I'll also keep the `aria-controls` attribute.